src/app/dim-ui/Dropdown.m.scss

```diff
@@ -55,3 +55,8 @@
   padding: 4px 8px;
   color: #999;
 }
+
+.seperator {
+  height: 1px;
+  background-color: #999;
```

Comment:
It's a bit bright, can we tone it down?

Reply:
<img width="793" alt="Screen Shot 2020-10-01 at 12 30 06 PM" src="https://user-images.githubusercontent.com/424158/94859933-2f8e0800-03ea-11eb-8520-a936703abd5f.png">

Reply:
<img width="583" alt="Screen Shot 2020-10-01 at 1 29 34 PM" src="https://user-images.githubusercontent.com/424158/94860217-9a3f4380-03ea-11eb-9eec-b7fbf8199819.png">